lib/chef/config.rb

```diff
@@ -569,12 +569,8 @@ def self.env
       ENV
     end
 
-    def self.windows_home_path
```

Comment:
Was this publicly accessible?
If so, i recommend we start with a deprecation warning before removing